RELEASE.md

```diff
@@ -0,0 +1,39 @@
+Release type: minor
+
+This release improves how we deal with custom scalars. Instead of being global
+they are now scoped to the schema. This allows you to have multiple schemas in
+the same project with differnet scalars.
```

Comment:
```suggestion
This release improves how we deal with custom scalars. Instead of being global
they are now scoped to the schema. This allows you to have multiple schemas in
the same project with different scalars.
```